code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -144,6 +149,9 @@
 	set category = "Spider"
 	set desc = "Spread a sticky web to slow down prey."
 
+	if(stat == UNCONSCIOUS)

```

Comment:
Changing to this make it worse, spider can still use abilities when dead.

As Fox said, there's a much more robust check.

The proc (function) incapacitated(), defined on the mob level (Find definition of a proc on visual studio code by typing proc/procname) and then expanded on (Might be the wrong technical term) in the mob/living level (Find it by /procname) has a much more robust check of:

``` /mob/living/incapacitated(ignore_restraints = FALSE, ignore_grab = FALSE, ignore_lying = FALSE)
	if(stat || paralysis || stunned || weakened || (!ignore_restraints && restrained()) || (!ignore_lying && lying))
		return TRUE
```

Which account for both type of stat (Either 1 or 2 will return true, only 0 (being conscious) will return false), and other conditions too.

Use that check instead of stat == UNCONSCIOUS, by calling the proc incapacitated().

Reply:
I asked how I should change it and got no specific reply, so thought "unconscious" also counted for being dead while checking any other type of incapacitation. I'll use this, instead.